hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/handlers/VolumeEntityHandler.java

```diff
@@ -63,12 +64,41 @@ public NamespaceSummaryResponse getSummaryResponse()
       totalKey += getTotalKeyCount(bucketObjectId);
     }
 
-    namespaceSummaryResponse.setNumTotalDir(totalDir);
-    namespaceSummaryResponse.setNumTotalKey(totalKey);
+    CountStats countStats = new CountStats(
+        -1, buckets.size(), totalDir, totalKey);
+
+    NamespaceSummaryResponse namespaceSummaryResponse =
+        NamespaceSummaryResponse.newBuilder()
+            .setEntityType(EntityType.VOLUME)
+            .setCountStats(countStats)
+            .setObjectDBInfo(getVolumeObjDbInfo(names))
+            .setStatus(ResponseStatus.OK)
+            .build();
 
     return namespaceSummaryResponse;
   }
 
+  private VolumeObjectDBInfo getVolumeObjDbInfo(String[] names)
+      throws IOException {
+    String dbVolumeKey = getOmMetadataManager().getVolumeKey(names[0]);
+    OmVolumeArgs volumeArgs =
+        getOmMetadataManager().getVolumeTable().getSkipCache(dbVolumeKey);
+    VolumeObjectDBInfo objectDBInfo = new VolumeObjectDBInfo();
```

Comment:
Same as `BucketObjectDBInfo`, create constructor or builder which takes `OmVolumeArgs` and create `VolumeObjectDBInfo` object.

Reply:
Thanks @hemantk-12 for review. Changes done. Pls re-review.